### ISSUE TITLE
load pretrained model cause error

### ISSUE THREAD

**Issue by AdanWang** [kosuke55/train_baiducnn#16]
when i tried  to load pretrained model from this website 'https://drive.google.com/uc?export=download&id=19IPtsVes3w-qogsiJToHmLrjCAdVEl9K'.I found that the struct of the net was wrong.It seems that you have changed the net.Could you please upload your new model?

Thanks a lot!

**Comment by kosuke55** [comment]
Sorry for the inconvenience. The model for that link is for the original_model branch. 
The master branch is currently in development, changes frequently and I haven't shared a trained model yet. Please be patient.

**Comment by AdanWang** [comment]
@kosuke55 thank you for your reply! I have trained the model with the Nuscene dataset. I found that the model was very easy to be overfit and did really bad performance in real data. Could you please tell me your model's accuracy so that i can figure out whether it is my mistake lead to the bad performance.
